src/main/java/com/amazonaws/encryptionsdk/jce/JceMasterKey.java

```diff
@@ -72,8 +72,8 @@
      */
     public static JceMasterKey getInstance(final SecretKey key, final String provider, final String keyId,
             final String wrappingAlgorithm) {
-        switch (wrappingAlgorithm) {
-            case "AES/GCM/NoPadding":
+        switch (wrappingAlgorithm.toUpperCase()) {
+            case "AES/GCM/NOPADDING":
```

Comment:
GitHub won't let me comment on lines that weren't changed in the PR, but we also need to apply this type of check on line 98 where we check the `RSA/ECB/` prefix.

Reply:
Added test case for the RSA/ECB prefix. This check is already applied